<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>allusers_images</class>
 <widget class="QDialog" name="allusers_images">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="push_request">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>147</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string>Request</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>159</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Request Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_back">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_user">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>108</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>661</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>108</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Images:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_views">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>470</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>611</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
